src/tooling/piral-cli-webpack5/src/enhancers/piral-instance-webpack-config-enhancer/README.md

```diff
@@ -0,0 +1,135 @@
+<div align="center">
```

Comment:
This file is potentially obsolete.

Reply:
I left the READEME files in as documentation. I don't have any stakes in that and can just remove them.